circle.yml

```diff
@@ -1,10 +1,15 @@
 version: 2
 executorType: machine
+docker:
+- image: circleci/python:3.6.4
 stages:
   build:
     workDir: ~/electricitymap
     steps:
       - type: checkout
+      - type: shell
```

Comment:
Do you really need to upgrade pip? Upgrading to latest version can always be problematic if the latest version breaks for some reason.
You can add the `pyenv global 3.5.2` shell command to force python3 by the way

Reply:
WHERE do I add `pyenv global 3.5.2`?  That is not working for me.

Yes.  Upgrading pip is the right thing to do.  The old pip was designed for Python 2 and a lot of important Python 3 related changes have happened in the past 18 months.  https://pip.pypa.io/en/stable/news/

Reply:
Gotcha.
For pyenv, just add it as a shell command as a first step (before upgrading pip)
```yaml
steps:
  - type: shell
    name: Set python version
    command: pyenv global 3.5.2
```